website/gatsby-node.js

```diff
@@ -192,3 +226,16 @@ function createDocPages(createPage, data) {
     });
   });
 }
+
+async function getGitLog(filepath) {
+  const logOptions = {
+    file: filepath,
+    n: 1,
+    format: {
+      date: `%ai`,
+      authorName: `%an`,
+    },
+  };
+
+  return await git().log(logOptions);
```

Comment:
`async/await` is not needed in this case, because the promise is immediately returned. Just return the Promise. It's more efficient in doing so, because it prevents from generating an unnecessary state machine for this particular function.